language/move-model/src/ty.rs

```diff
@@ -80,16 +81,16 @@ impl PrimitiveType {
         }
     }
 
-    /// Attempt to convert this type into a language_storage::TypeTag
-    pub fn into_type_tag(self) -> Option<TypeTag> {
+    /// Attempt to convert this type into a language_storage::Type
+    pub fn into_type(self) -> Option<MType> {
```

Comment:
I would suggest to change this to `into_normalized_type` for consistency

Reply:
Updated!